src/components/overview/sub-Components/BasicDetails.jsx

```diff
@@ -20,9 +20,9 @@ const BasicDetails = function(props) {
     return (
       <section class="section">
         <div class="container">
-          <div>{props.state.basicDetails.category}</div>
-          <div>{props.state.basicDetails.name}</div>
-          <div>{prices}</div>
+          <h5>{props.state.basicDetails.category}</h5>
```

Comment:
Posssibly consider using object destructuring in your function parameters to have less lengthy variable usage (i.e. "props.state.basicDetails.category")